scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+    example: Saguinus oedipus
+  ref_system:
+    label: Projection
+    description: Reference system for the area of interest
+    type: int
+    example: 3116
+  elev_margin:
+    label: Elevation buffer
+    description: Magnitude of buffer to add to the elevation ranges of the species (usualy in meters), if no buffer then 0
+    type: int
+    example: 0
+  forest_threshold:
+    label: Tree cover percentage 
+    description: 
+      Percentage of tree cover of reference for the Global Forest Watch data according to the tolerance of the species. 
+      This value is given according to suitable habitat of the species. If not available, use Map of Life Values 
+      (e.g. [https://mol.org/species/range/Saguinus_oedipus]) 
+    type: int
+    example: 50
+  country_code:
+    label: Country code
+    description: Three letter code (ISO_3166-1_alpha-3) for the country where the analysis will be done.
+    type: text
+    example: COL
+  region:
+    label: Region
+    description: Name of the region according to the codification from the GADM data [https://gadm.org/metadata.html] for the variable NAME_1
+    type: text
+    example: ""
+  spat_res:
+    label: Spatial resolution
+    description: Spatial resolution for the analysis, in meters
+    type: int
+    example: 1000
+  expert_source:
+    label: Expert range source
+    description: Source of the expert range map
+    type: options
+    options:
+      - MOL
+      - IUCN
+      - QC
+    example: IUCN
+  t_0:
+    label: Initial time
+    description: Year where the analysis should start
+    type: int
+    example: 2000
+  t_n:
+    label: Final time
+    description: Year where the analysis should end (it should be later than Initial time).
+    type: int
+    example: 2020
+  time_step:
+    label: Time step
+    description:
+      Time range between each time step. It can be one to get change every year or two if the change 
+      is going to be evaluated every two years and so on.
+    type: int
+    example: 1
+outputs:
+  df_shi:
+    label: SHI values
+    description: This TSV (Tab Separated Values) data shows the values of the SHI according to the parameters entered.
+    type: text/tab-separated-values
+#    example: /Saguinus oedipus_SHI_table.tsv
+  img_shi_time_period:
+    label: SHI for the selected period
+    description: Figure with the map of changes through the years selected to calculate the SHI.
+    type: image/png
+ #   example: /Saguinus oedipus_GFW_change.png
+  img_shi_timeseries:
+    label: Time series of the SHI and score values for the selected period
+    description: Figure with the time series of the years selected and the scores to calculate SHI.
+    type: image/png
+ #   example: /Saguinus oedipus_SHI_timeseries.png
+  r_habitat_by_year:
+    label: Raster of available habitat by year
+    description: Raster with 1 where there is available habitat for the species and leaves the rest empty.
+    type: image/tiff;application=geotiff
```

Comment:
It seems you are saving all years in on file with multiple bands (I think layers names are not preserve here)
If you want to save each year just add [] at the end.  (remember to preserve the name of the layers that are informative in the .tif file)